jetty-client/src/main/java/org/eclipse/jetty/client/Origin.java

```diff
@@ -48,6 +60,11 @@ public Address getAddress()
         return address;
     }
 
+    public Object getTag()
```

Comment:
Is javadoc needed here to indicate that this is the same tag from the associated request?